mkt/search/forms.py

```diff
@@ -71,7 +72,10 @@
     ('featured-game-adventure', 'featured-game-adventure'),
     ('featured-game-puzzle', 'featured-game-puzzle'),
     ('featured-game-strategy', 'featured-game-strategy'),
+    ('featured-website', 'featured-website'),
 ]
+REG_TAG = lambda r: ('featured-website-%s' % r, 'featured-website-%s' % r)
```

Comment:
don't know if we need an intermediate lambda, it might be more readable without it, although i understand what it's doing.

```[('featured-website-%s' % r, featured-website-%s' % r) for r in dict(REGION_CHOICES.keys())]